clustering.Rmd

```diff
@@ -320,64 +394,92 @@ in the fourth iteration; both the centers and labels will remain the same from t
 > ways to assign the data to clusters. So at some point, the total WSSD must stop decreasing, which means none of the assignments
 > are changing, and the algorithm terminates.
 
+What kind of data is suitable for K-means clustering? 
+In the simplest version of K-means clustering that we have presented here,
+the straight-line / Euclidean distance is used to measure the 
```

Comment:
I would just put "straight-line distance"  -- we only mention in passing that it's also called "Euclidean" when we first introduce it, but refer to it everywhere else as strgiht-line

Reply:
fixed